_gsocprojects/2020/proposal_ScienceBoxMonitor.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
This proposal file should be in `_gsocproposals/2020` directory…

Reply:
It is fixed now, sorry